src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -1280,16 +1281,16 @@ static public int endOfShard(int shardIndex, int shardSize) {
     }
 
     /**
-     *  Same as {@link IntervalUtils::combineBreakpoints}, but sorts the inputs first.  Sorted versions are kept in a new list.
+     *  Same as {@link IntervalUtils::combineAndSortBreakpoints}, but sorts the inputs first.  Sorted versions are kept in a new list.
```

Comment:
Sorry, maybe I was unclear in the last review---I don't think this method is necessary, right?  Isn't it pretty much equivalent to `combineAndSortBreakpoints`?  (And if not, the method names don't make clear the difference between the two.)  You will need to make that method public, though.